packages/hub/node-tests/routes/checkly-webhook-test.ts

```diff
@@ -0,0 +1,42 @@
+import { registry, setupHub } from '../helpers/server';
+
+class StubStatuspageApi {
+  async createIncident(_componentName: string, _name: string) {
+    return true;
+  }
+  async resolveIncident(_componentName: string) {
+    return true;
+  }
+}
+
+describe('POST /api/checkly-webhook', async function () {
+  this.beforeEach(function () {
+    registry(this).register('statuspage-api', StubStatuspageApi);
+  });
+
+  let { request } = setupHub(this);
+
+  it('returns 200 when processing a check', async function () {
+    await request()
+      .post('/api/checkly-webhook')
+      .set('Accept', 'application/vnd.api+json')
+      .set('Content-Type', 'application/vnd.api+json')
```

Comment:
it would be really strange if checkly actually called us with a JSON-API content type. I'm pretty sure they use `application/json`.